stories/notifications.stories.js

```diff
@@ -185,36 +185,6 @@ storiesOf( 'Global/Notifications', module )
 			type="win-stats-increase"
 		/>
 	) )
-	.add( 'Publishing Win', () => (
-		<Notification
-			id="notification-id"
-			title={ __( 'Congrats on five published posts', 'google-site-kit' ) }
-			description={ __( 'That’s out of this world. Here are the combined stats for your posts', 'google-site-kit' ) }
-			dismiss={ __( 'OK, Got it!', 'google-site-kit' ) }
-			format="large"
-			winImage={ rocketImage }
-			blockData={
-				[
-					{
-						title: 'Total Views',
-						datapoint: 413,
-						datapointUnit: '',
-					},
-					{
-						title: 'Average Impressions',
-						datapoint: 735,
-						datapointUnit: '',
-					},
-					{
-						title: 'Average CTR',
-						datapoint: 12.9,
-						datapointUnit: '%',
-					},
-				]
-			}
-			type="win-stats"
-		/>
-	) )
 	.add( 'Total Stats', () => (
```

Comment:
There are a few more stories to remove here. Most of the ones below here are also publisher wins - they basically replicate the UI the publisher wins code removed would have resulted in. (Probably the production logic was too complicated to rely on for Storybook - good to get that removed finally. 😆)